hieradata/dev/wso2/common.yaml

```diff
@@ -51,10 +51,14 @@ wso2::template_list:
   - bin/wso2server.sh
 #  - bin/ciphertool.sh
 
-# Puppet file list to be copied
+# Puppet file list to be copied triggering restart
 #wso2::file_list:
 #  - repository/conf/security/cipher-tool.properties
 #  - repository/conf/security/cipher-text.properties
+#  - repository/conf/security/secret-conf.properties
+
+# Puppet file list to be copied without restarting the service
```

Comment:
Isn't this giving a different meaning? Wouldn't it be more appropriate to say "Files to be copied when restarting the system service"? In some other location we might need to mention that the system service is only restarted if there are distribution changes or configuration changes available.